jekyll/_cci2/ip-ranges.md

```diff
@@ -59,7 +59,7 @@ workflows:
 ## List of IP address ranges associated with the IP ranges feature
 {: #listofipaddressranges }
 
-*Last updated*: 08/02/2021
+*Last updated*: 08/23/2021
```

Comment:
Could we change this date format to be ISO 8601 to remove ambiguity about its meaning? Only part of the world uses slashes or the month-day ordering, but ISO 8601 is standard across the computing industry.

```suggestion
*Last updated*: 2021-08-23
```